src/connectedmachine/azext_connectedmachine/aaz/latest/connectedmachine/private_link_scope/_update.py

```diff
@@ -15,10 +15,10 @@
     "connectedmachine private-link-scope update",
 )
 class Update(AAZCommand):
-    """Update an Azure Arc PrivateLinkScope. Note: You cannot specify a different value for InstrumentationKey nor AppId in the Put operation.
+    """Update an Azure Arc PrivateLinkScope. Note: You cannot                                          specify a different value for InstrumentationKey nor AppId in the Put operation.
```

Comment:
Why did you add these spaces?

Reply:
Not sure what happens when using aaz, but fixed.